test/account.spec.ts

```diff
@@ -196,112 +379,16 @@ describe('privateToPublic', function() {
   it('should produce a public key given a private key', function() {
     const pubKey =
       '3a443d8381a6798a70c6ff9304bdc8cb0163c23211d11628fae52ef9e0dca11a001cf066d56a8156fc201cd5df8a36ef694eecd258903fca7086c1fae7441e1d'
-    const privateKey = Buffer.from([
-      234,
-      84,
-      189,
-      197,
-      45,
-      22,
-      63,
-      136,
-      201,
-      58,
-      176,
-      97,
-      87,
-      130,
-      207,
-      113,
-      138,
-      46,
-      251,
-      158,
-      81,
-      167,
-      152,
-      154,
-      171,
-      27,
-      8,
-      6,
-      126,
-      156,
-      28,
-      95,
-    ])
+    // prettier-ignore
+    const privateKey = Buffer.from([234, 84, 189, 197, 45, 22, 63, 136, 201, 58, 176, 97, 87, 130, 207, 113, 138, 46, 251, 158, 81, 167, 152, 154, 171, 27, 8, 6, 126, 156, 28, 95,])
```

Comment:
Unrelated to this PR but I think for the form it would be better to convert this to a hex string to be consistent.

Reply:
done in https://github.com/ethereumjs/ethereumjs-util/pull/275/commits/6c89111e4b60b4fd3a1d98205199d6d274b61959